### DISCUSSION TITLE
设置密码一直报错404

### DISCUSSION THREAD

**Discussion by like977** [spencerwooo/onedrive-vercel-index#634] [General]
![image](https://user-images.githubusercontent.com/43336441/166970369-7f0aac57-39f7-4ce9-a454-22645a2f52c0.png)
![image](https://user-images.githubusercontent.com/43336441/166970413-f5dd7de6-9cf8-45a2-aa55-459eab7587db.png)
![image](https://user-images.githubusercontent.com/43336441/166970536-4171a6ef-9774-4d23-88f2-08f8da258fc4.png)

设置密码一直报错404

**Comment by myl7** [comment]
密码是纯数字吗？如果是的话可以参考 #638 修复。
